packages/web-api/src/methods.ts

```diff
@@ -1225,7 +1225,7 @@ export interface ChatPostMessageArguments extends WebAPICallOptions, TokenOverri
 export interface ChatScheduleMessageArguments extends WebAPICallOptions, TokenOverridable {
   channel: string;
   text?: string;
-  post_at: string;
+  post_at: string | number;
```

Comment:
One thought: by not limiting it to only `number`, there's a greater possibility that folks will continue to interpret `string` as possibly being a timestamp, as the dev in this case did. By switching it to `number`, we reduce the chance of that happening.

Reply:
@misscoded As I mentioned in this PR's description and my response to the developer, using `string` here is totally fine. It works. The issue the developer had was that they used non-integer value. For this reason, we still can have `string` as an acceptable type for it. If we remove `string` from here, it's a breaking change.